### PR TITLE
add toggle function to bookmark

### PR DESCRIPTION
**Pull request by AnnaD91**
add JS script to html 
create toggle function to first bookmark button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,7 +33,12 @@ <h2 class="card__question">
               <li class="card__tag-list-item">#english voices</li>
             </ul>
             <div class="card__button-bookmark">
-              <button class="bookmark" aria-label="bookmark" type="button">
+              <button
+                class="bookmark"
+                data-js="button-bookmark"
```
Comment (B-Baumeister) [inline index.html:42]: looks good =)

**Review by B-Baumeister — APPROVED** [review]
LGTM =)

**Comment by PZhu888** [comment]
super!

File: index.js [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
 
 bookmarkButton.addEventListener("click", () => {
   bookmarkButton.classList.toggle("bookmark--active");
 });
+
+answerButton.addEventListener("click", () => {
+  cardAnswer.classList.toggle("card__answer--active");
+  cardAnswer.classList.toggle("hidden");
+
+  if (cardAnswer.classList.contains("hidden")) {
```
Comment (PZhu888) [inline index.js:16]: nice job!

**Review by B-Baumeister — APPROVED** [review]
suppii =)

**Pull request merged**